bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -89,22 +89,48 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: CertificateRequest
+    - description: "A CertificateRequest is used to request a signed certificate from\
+        \ one of the configured issuers. \n All fields within the CertificateRequest's\
+        \ `spec` are immutable after creation. A CertificateRequest will either succeed\
+        \ or fail, as denoted by its `status.state` field. \n A CertificateRequest\
+        \ is a one-shot resource, meaning it represents a single point in time request\
+        \ for a certificate and cannot be re-used."
+      displayName: CertificateRequest
+      kind: CertificateRequest
       name: certificaterequests.cert-manager.io
       version: v1
-    - kind: Certificate
+    - description: "A Certificate resource should be created to ensure an up to date\
+        \ and signed x509 certificate is stored in the Kubernetes Secret resource\
+        \ named in `spec.secretName`. \n The stored certificate will be renewed before\
+        \ it expires (as configured by `spec.renewBefore`)."
+      displayName: Certificate
+      kind: Certificate
       name: certificates.cert-manager.io
       version: v1
-    - kind: Challenge
+    - description: Challenge is a type to represent a Challenge request with an ACME
+        server
+      displayName: (Internal) Challenge
+      kind: Challenge
       name: challenges.acme.cert-manager.io
       version: v1
-    - kind: ClusterIssuer
+    - description: A ClusterIssuer represents a certificate issuing authority which
+        can be referenced as part of `issuerRef` fields. It is similar to an Issuer,
+        however it is cluster-scoped and therefore can be referenced by resources
+        that exist in *any* namespace, not just the same namespace as the referent.
+      displayName: ClusterIssuer
+      kind: ClusterIssuer
       name: clusterissuers.cert-manager.io
       version: v1
-    - kind: Issuer
+    - description: An Issuer represents a certificate issuing authority which can
+        be referenced as part of `issuerRef` fields. It is scoped to a single namespace
+        and can therefore only be referenced by resources within the same namespace.
+      displayName: Issuer
+      kind: Issuer
       name: issuers.cert-manager.io
       version: v1
-    - kind: Order
+    - description: Order is a type to represent an Order with an ACME server
+      displayName: (Internal) Order
```

Comment:
I got the idea for prefixing these with `(Internal)` from the CouchDB operator

https://github.com/k8s-operatorhub/community-operators/blob/81a8e860d17591a6acb946a581da4c669ca142d0/operators/couchdb-operator/2.2.1/manifests/couchdb-operator.clusterserviceversion.yaml#L147-L152